x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/timeline/timeline.tsx

```diff
@@ -56,6 +56,13 @@ const i18nTexts = {
   hotPhase: i18n.translate('xpack.indexLifecycleMgmt.timeline.hotPhaseSectionTitle', {
     defaultMessage: 'Hot phase',
   }),
+  rolloverTooltip: i18n.translate(
+    'xpack.indexLifecycleMgmt.timeline.hotPhaseRolloverToolTipContent',
+    {
+      defaultMessage:
+        'Data age is only considered after rollover has occurred which adds variation to time in hot phase. See hot phase settings.',
```

Comment:
I think this could be clearer. It seems like you're trying to make two different points: 

1. The min_age for moving to the next phase is based on the elapsed time since  rollover, not the absolute age of the data. 
2. The total time spent in the hot phase can vary depending on what triggers the rollover. 

That's a lot to pack into a tooltip. :-) Maybe:

```suggestion
        'How long it takes to reach the rollover criteria in the hot phase can vary. Data moves to the next phase when the time since rollover reaches the minimum age.',
```